owners/info_server.js

```diff
@@ -14,7 +14,14 @@
  * limitations under the License.
  */
 
+const fs = require('fs');
 const express = require('express');
+const _ = require('lodash');
+const hl = require('highlight').Highlight;
```

Comment:
Probot uses [handlebars](https://handlebarsjs.com/) templates built-in, so you can consider using those instead of highlight and reduce the number of dependencies used here (although if you do plan on switching to handlebars, you should still explicitly add it as a dependency)

Reply:
I'm not sure I understand what you're saying here. Highlight provides syntax-highlighting of the OWNERS file (rendering into HTML with classes). Lodash is a part of Probot and it contains templates built-in, so this is doing the same thing re: using an existing dependency.